src/app/layout/footer/footer.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import { SocialLink  } from '../../shared/models/social-link.component';
```

Comment:
Separate your imports according to the Angular style guide: https://angular.io/guide/styleguide#import-line-spacing Angular imports first, external imports then, and local imports last. Each one separated by a blank line:
```
import { Component, OnInit } from '@angular/core';

import { SocialLink } from '../../shared/models/social-link.component';
```